code/model/UserDefinedForm.php

```diff
@@ -1120,6 +1141,18 @@ public function finished() {
 			'Form' => '',
 		));
 	}
+	
+	// Encrypt Function
+	public function mc_encrypt($encrypt){
+	    $encrypt = serialize($encrypt);
+	    $iv = mcrypt_create_iv(mcrypt_get_iv_size(MCRYPT_RIJNDAEL_256, MCRYPT_MODE_CBC), MCRYPT_DEV_URANDOM);
```

Comment:
You explicitly enforce MCRYPT_RJINDAEL_256. This should be left up to the dev, who may want to use the standard AES-128. As above, place the cipher mode as a private static variable, so that it can be set via Config.